### PR TITLE
Using Bootstrap code review request.

### PR DESCRIPTION
**Pull request by Meltrust**
# In this project I:

- Built a clone from the "Newsweek" webpage using _HTML,_  _Bootstrap,_ and _CSS_.

- Added Bootstrap classes to align almost everything in order to make the page usable in devices of all sizes without the use of custom media queries.

- Put emphasis on how it behaves differently depending on the size of the screen where the website is rendered using Bootstrap breakpoints classes.

- Made extensive use of the framework.

- Built a descriptive readme for the project.

- Set up a code linter.

- Issued a descriptive pull request.

- Worked alone on this project.

### CODE REVIEW

File: index.html
Comment (adaorachi) [inline index.html]: - [x] The subscription and signup buttons should contain links. So ensure they are wrapped in anchor tags.

File: index.html
Comment (adaorachi) [inline index.html]: - [x] Ensure you have your navigation bar items are links. So they should be enclosed in anchor tags.

**Comment by adaorachi** [comment]
👋 Hi @Meltrust,

Well done on the completion of the first draft of this project. You did a pretty nice job on it and I really commend that. Your images and contents are well-positioned and aligned. You made adequate use of the Bootstrap classes and also have a well-written Pull Request summary. 👍 💥 

However, there are a few changes I would want you to implement before your project can get an approval.
So, hang in there, you are just a step away from completing this project. 💪

---
### **CHANGES REQUIRED** ♻️ 
---

- [x] Your Project view on the browser has a white trailing space by the right. The overflow occurs in the medium breakpoint between 576px and 730px. And this can be noticed with the scrolling bar highlighted in the screenshot below. Kindly fix the styling of the elements causing the overflow. 

- [x] Also ensure your navigation bar is well padded by the sides and has less space on the top and bottom. See the screenshots below and also see the original site for better reference.

- [x] Also ensure to include the tags at the right bottom of every image.

- [x] The title '**More stories**' is misnamed with a different title. It should have the '**Featured Stories**' title instead. Actually, you have two titles misnamed in two different sections. Find the other misnamed title in the middle section of your project page and kindly fix them.

**Your Project Version:**

boot1

**Original Version**

boot11

---

- [x] In the screenshots below, there you have the other misnamed section I mentioned in the previous comment. Kindly change it to 'More Stories', just like you have it in the original site. And thereafter adjust the image sizes and contents to look similar to the original. Attached below are screenshots for reference. Also, see the original site should you need more clarification.

**Your Project Version:**

boot33

**Original Version:**
boot3

---

- [x]  Here, in the small breakpoint, the '**In the magazine**' section contents should span the entire width of the page. See the screenshots below for reference. Endeavor to make reference to the original site too.

**Your Project Version**

boot6

**Original Version:**

boot66

---

- [ ] In the section just below the '**Editor's pick**', where it has other sections of interest with titles 'US', 'WORLD', ...etc, ensure the contents are properly aligned in **small breakpoints**. The two large images should span the whole widths of the page while the aside smaller images are positioned below the larger images in a column as well. It should look similar to that of the original page.

**Your Project Version:**

boot7

**Original Version** 

boot77


---

- [x] Here, the 'Culture & Travel' section should span the whole width in **small breakpoints**. Kindly view the original site for more reference.

**Your Project Version**

boot8

**Original Version**

boot88

---

- [x] Here, you are missing the contents below the section highlighted in the screenshot below. It should also have more article contents with bordered texts.  See the highlighted area in the screenshot and also compare it to that of the original site to see differences.

**Your Project Version:**

boot9

**Original Version**

boot99

---

File: index.html
Comment (adaorachi) [inline index.html:23]: - [ ] Here, kindly delete the commented codes. If you need to review a previous version of your code, you can make use of git history.

**Review by adaorachi — CHANGES_REQUESTED** [review]

File: index.html
Comment (adaorachi) [inline index.html]: - [x] Here, kindly delete the commented codes. If you need to review a previous version of your code, you can make use of git history.

**Review by adaorachi — CHANGES_REQUESTED** [review]

File: index.html
Comment (adaorachi) [inline index.html]: - [Optional] Here it is the usual and proper convention to have your title just below the meta tags, and I would encourage you to adhere to it.

**Comment by adaorachi** [comment]
---

**The following requirements are considered OPTIONAL and so, are NOT mandatory but I will encourage you to take a moment to look into it as it will be helpful to you in this project and future ones.
NOTE: Requirements NOT marked as OPTIONAL are mandatory!**

---

- [Optional] In your CSS file, I can see you made good use of the bootstrap classes in your HTML but there are still a couple of them which can be incorporated still with bootstrap. For example, take a look at the max-width: 1280px - that can be achieved with the container-fluid bootstrap class. Background-color: #000 can have the class '**bg-dark**'. Your '**list-style: none**' can also be achieved with bootstrap classes - '**list-item**',  which removes all default list styling and make them look better.

There are also bootstrap classes for margin-left, width: 100%, flex-grow, position, padding, and also a ton of them. I encourage you to look in them and get a better grasp of it for future projects.

---

**Comment by adaorachi** [comment]
### **After implementing the MANDATORY changes highlighted above, request for another review by submitting a Code Review Form.**

Should you have any questions or need any help, please feel free to reach out to me on Slack(@adaorachi). ☺️ 

**Happy Coding and Good-luck on it.**

**MaryAnn Chukwuka**

**Comment by Meltrust** [comment]
Ty for the review!!! I did everything as indicated..  Requesting another code review.

**Comment by lekegitrepo** [comment]
#### Hi @Meltrust :+1:
#### You did a good job :muscle: by implementing the required changes mentioned in the previous review. However, you need to fix the linter errors and also fix the consistency of the sign in and subscribe button of the navbar at the mobile screen: 425px.

**_kindly fix these and submit for review._**

### _Reviewed by Temitayo Adeleke._

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
#### Status: Request changes.

**Comment by Meltrust** [comment]
Done! (linter error was a server error according to their website).

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @Meltrust, you did a great job fixing all the changes requested by the previous TSE's :+1: 
Go ahead and merge to master, do not forget to submit the project completion form.

Good luck with your next project
_Happy coding_ :man_technologist: 
Daniel Larbi Addo on slack

**Pull request merged**
